src/app/settings/settings.page.ts

```diff
@@ -74,26 +73,9 @@ export class SettingsPage {
     const str = property.key + '=' + property.value;
     this.logger.debug('Sending: ' + str);
```

Comment:
**Suggestion:** It's better to use a more descriptive name for the 'str' variable.
```suggestion
    const propertyString = property.key + '=' + property.value;
    this.logger.debug('Sending: ' + propertyString);
```